deploy-0chain/action.yml

```diff
@@ -447,6 +447,28 @@ runs:
           echo "==========================================================="
         fi
 
+    - name: "Install Zs3Server"
```

Comment:
minor: rename to match what this step does

Reply:
Done